CSS/style.css

```diff
@@ -428,6 +431,13 @@ footer {
   background-color: var(--white);
 }
 
+/* contact-form */
+
+form {
+  margin-top: 2rem;
+}
```

Comment:
- [ ] Please make sure you're using `flexbox` to layout the elements as specified in the requirements ✔.